.devcontainer/setup.py

```diff
@@ -71,7 +71,6 @@ def install_tools():
     """Installs Go developer tools."""
 
     run_commands([
-        "curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh",
         "curl https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /root/go/bin",
         "curl https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b /root/go/bin",
```

Comment:
It seems that this also can be replaced with `go install -modfile=tools/go.mod ...`

Reply:
Should I merge it? And add 
```
 github.com/go-delve/delve/cmd/dlv@latest \
                golang.org/x/tools/gopls@latest \
                mvdan.cc/gofumpt@latest \
                mvdan.cc/gofumpt/gofumports"
```
also to tools ?

Reply:
Better to install everything that possible with `tools`, there we can easily control versions.

Reply:
We can remove `run_commands` and install all that we need with `make init` as we did it in saas repos.

https://github.com/percona-platform/authed/blob/7c0fcc523941a56d8d337f21c5e8d0175864805b/Makefile#L11-L16

Reply:
CI failed:
`/usr/local/go/pkg/tool/linux_amd64/link: running gcc failed: exec: "gcc": executable file not found in $PATH`

Reply:
The problem already solved as I see.